### ISSUE TITLE
Error when trying to export files - Technician profile with not enabled "See my tickets"

### ISSUE THREAD

**Issue by nuria2c** [glpi-project/glpi#14144]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.6

### Bug description

When the technician profile with the capability "See My tickets" (Assistance > Tickets) uses the functionality to export files from the tickets search screen, receives the message “You don't have permission to perform this action”.

We think that is a global permission issue because needs the permisson access to "own tickets".

This isse may be can be related to:
https://github.com/glpi-project/glpi/issues/12359

By the other side, if you have the rights and export directly the dtat without click the "Search" button the export includes all the record. Not only the showed once.

### Page URL

/front/ticket.php

### Steps To reproduce

1. Go to Suport > Tickets with a technician profile with this capability disabled
2. Use the export tool
3. Select "Export all to csv" or any other option
4. The system returns "You don't have permission to perform this action."

### Your GLPI setup information

GLPI 10.0.6 ( => /var/www/html/glpi)
Installation mode: TARBALL
Current language:en_GB

**Comment by nuria2c** [comment]
May be also can be related with this other isse we opened some days ago
https://github.com/glpi-project/glpi/issues/13904

**Comment by msizec** [comment]
Im facing the same bug

I did find this error in php-error.log after trying to export tickets
`
[2023-03-14 17:32:47] glpiphplog.CRITICAL:   *** PHP Error (1): Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes) in /var/www/glpi/vendor/tecnickcom/tcpdf/tcpdf.php at line 16678`

**Comment by trasher** [comment]
You should try to augment your PHP memory_limit

**Comment by msizec** [comment]
The error message is wrong, we do not have memory issue, I can reproduce the problem and the memory isn't fully used though.
I tried to reproduce when on the tickets page with 9 tickets, can't reproduce using the different export options.
Tried on a tickets page with 46 tickets, option set to limit view to 1000 items, could reproduce it using the export all pages to PDF : page is loading continuously a few minutes then fails with a 500 error page.
Tried a second time using the "export this page to csv" : page was loading a few minutes and my file was created : a csv file with 1000 rows => Like OP said, there is a problem with rights. My file should only contain 46 rows.
